core/dbt/events/types.py

```diff
@@ -11,6 +11,11 @@
 from dbt.events.base_types import WarnLevel, InfoLevel, DebugLevel, ErrorLevel, DynamicLevel
 
 
+# TODO Move this to dbt_common.ui
+def _error_tag(msg: str) -> str:
+    return f'[{red("ERROR")}]: {msg}'
```

Comment:
Are you going to do this TODO before merging?

Reply:
I do not. That would entail opening a a PR in dbt-common and also getting it released, and I didn't want that to block this PR

Reply:
Follow up tasks assigned to me:
* https://github.com/dbt-labs/dbt-core/issues/9914
* https://github.com/dbt-labs/dbt-common/issues/107